QATCH/processors/Elaborate.py

```
@@ -205,35 +220,44 @@ def savitzky_golay(self, y, window_size, order, deriv=0, rate=1):
         from math import factorial
 
         import numpy as np
+
         try:
             window_size = np.abs(int(window_size))
             order = np.abs(int(order))
         except ValueError as msg:
-            raise ValueError(
-                "WARNING: window size and order have to be of type int!")
+            raise ValueError("WARNING: window size and order have to be of type int!")
```

Ruff (0.15.13)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)